backend/migrations/2023-06-02-203505_layers_relation_table/up.sql

```diff
@@ -0,0 +1,9 @@
+-- Your SQL goes here
+CREATE TYPE LAYER_TYPE AS ENUM ('base', 'soil', 'todo', 'label', 'paths', 'photo', 'shade', 'trees', 'winds', 'zones', 'plants', 'drawing', 'terrain', 'habitats', 'warnings', 'watering', 'landscape', 'hydrology', 'fertilization', 'infrastructure');
+CREATE TABLE layers (
+    id SERIAL PRIMARY KEY,
+    map_id INTEGER REFERENCES maps(id) NOT NULL,
+    type LAYER_TYPE NOT NULL,
+    name TEXT NOT NULL,
```

Comment:
It was simply a case of simplicity for me. Instead of having a check in the frontend and then dynamically generate the display name from the layer type, I thought it would make sense to do this once during map creation and then not have to worry in the frontend.